srv/GetVscSettings.srv

```diff
@@ -0,0 +1,5 @@
+---
+bool srv_ready
+string serial_number
+string firmware_version
+int16 radio_power_level #in dB
```

Comment:
```suggestion
int16 radio_power_level_db
```
I like that you added the comment. In thinking about it more and seeing how it is used int he other PR, can you please add the `_db` where ever that `radio_power_level` is used.

Reply:
Decided to use `radio_power_db` for shortness.